src/foam/u2/view/ReadReferenceView.js

```diff
@@ -48,58 +137,123 @@ foam.CLASS({
-              .end();
+              return this.E()
+                .start('a')
+                  .attrs({ href: '#' })
+                  .on('click', (evt) => {
+                    evt.preventDefault();
+                    if ( self.linkTo === 'daoSummary' ) {
+                      const pred = foam.mlang.predicate.False.create();
+
+                      self.stack.push(
+                        {
+                          class: 'foam.comics.v2.DAOSummaryView',
+                          data: self.obj,
+                          of: self.obj.cls_,
+                          backLabel: 'Back',
+                          config: self.DAOControllerConfig.create({
+                            daoKey: self.prop.targetDAOKey,
+                            createPredicate: pred,
+                            editPredicate: pred,
+                            deletePredicate: pred,
+                            editEnabled: false
+                          })
+                        },
+                        self
+                      );
+                      // link to a menu
+                    } else {
+                      self.pushMenu(self.linkTo);
+                    }
+                  })
+                  .tag(self.ReferenceCitationView, { data: obj })
+                .end();
             } else {
-              return this.E().start()
-                .tag(self.ReferenceCitationView, {data: obj})
-              .end();
+              return this.E()
+                .start()
+                  .tag(self.ReferenceCitationView, { data: obj })
+                .end();
             }
-          }));
+          })
+        );
       }
     },
 
     function fromProperty(prop) {
       this.SUPER(prop);
+
       this.prop = prop;
-      var dao = this.ctrl.__subContext__[prop.targetDAOKey];
-      this.permissionEnableLinkCheck().then(() => {
-        if ( dao )
-          dao.find(this.data).then((o) => this.obj = o);
+
+      // set link config properties
+      // first figure out where these properties were provided (i.e., set to reference property or passed to this view)
+      this.enableLink = this.prop.enableLink && this.enableLink;
+      this.menuKeys = this.prop.menuKeys || this.menuKeys;
+
+      this.configLink().then(() => {
+        const dao = this.ctrl.__subContext__[prop.targetDAOKey];
+        if ( dao ) {
+          dao.find(this.data).then((o) => (this.obj = o));
+        }
       });
     },
 
-    async function permissionEnableLinkCheck() {
-      if ( ! this.auth ) return;
-      let permission = `${this.prop.of.id}.${this.prop.name}.disableRefLink`;
-      permission = permission.toLowerCase();
-      await this.auth.check(this.__subContext__, permission).then( check => {
-        this.enableLink = ! check;
-      })
+    async function configLink() {
+      // enableLink set to false?
+      if ( ! this.enableLink ) {
+        this.enableLink = false;
```

Comment:
test condition for this line is what this line is setting. If false then set to false - is seemingly unnecessary.

Reply:
reason for this was explained https://github.com/kgrgreer/foam3/pull/350/files#r689788330